crest/Assets/Crest/Crest/Scripts/OceanChunkRenderer.cs

```diff
@@ -160,10 +160,10 @@ public static void ExpandBoundsForDisplacements(Transform transform, ref Bounds
         {
             float boundsPadding = OceanRenderer.Instance.MaxHorizDisplacement;
             float expandXZ = boundsPadding / transform.lossyScale.x;
-            float boundsY = OceanRenderer.Instance.MaxVertDisplacement / transform.lossyScale.y;
+            float boundsY = OceanRenderer.Instance.MaxVertDisplacement;
             // extend the kinematic bounds slightly to give room for dynamic sim stuff
-            boundsY = Mathf.Max(boundsY, 1f);
-            bounds.extents = new Vector3(bounds.extents.x + expandXZ, boundsY, bounds.extents.z + expandXZ);
+            boundsY += 5f;
```

Comment:
This isn't feedback, but just curiousity about why the logic here was changed.

Reply:
i've increased the bounds a bit - tiles were getting culled too aggressively